spec/models/control_spec.rb

```diff
@@ -0,0 +1,22 @@
+RSpec.describe Control, type: :model do
+  let(:control) { build(:control) }
+
+  describe "validations" do
+    describe "boost_amount" do
```

Comment:
If you're testing this, it would seem to imply a precedent of checking the presence of name?